Formula/maxwell.rb

```diff
@@ -21,14 +21,14 @@ def install
       bin.install libexec/"bin/#{f}"
     end
 
-    bin.env_script_all_files(libexec/"bin", Language::Java.java_home_env("1.8"))
+    bin.env_script_all_files(libexec/"bin", Language::Java.java_home_env("11.0"))
   end
 
   test do
     fork do
-      exec "#{bin}/maxwell --log_level=OFF > #{testpath}/maxwell.log 2>/dev/null"
+      exec "#{bin}/maxwell --help > #{testpath}/maxwell.log 2>/dev/null"
```

Comment:
Please actually run maxwell normally and check the output from that. It found this error, but it might not find more delicate ones.

Reply:
A normal run requires a properly configured MySQL instance, is that something we want to do in a formula's test script?

Running with no args on my machine (which does have a properly configured instance) gives:

```
❯ maxwell
Using kafka version: 1.0.0
14:30:47,518 WARN  MaxwellConfig - maxwell mysql host not specified, defaulting to localhost
14:30:48,858 ERROR Maxwell - SQLException: Access denied for user ''@'localhost' (using password: NO)
```

It seems undesirable to attempt to connect to the local machine's MySQL instance (if any) in a test, but I can put something together that checks for that specific message (perhaps the warning) if you think that's better than what I've got right now.

Reply:
I suppose I could configure it to connect to a bogus host to avoid connecting to a users actual running MySQL instance (if any). Then have the test check for that specific error.

```
❯ maxwell --host bogus.host
Using kafka version: 1.0.0
14:32:36,868 ERROR Maxwell - SQLException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
```

Reply:
That seems like a good solution

Reply:
Done!